FF1Lib/Flags.cs

```diff
@@ -251,6 +252,8 @@ public class Flags : IIncentiveFlags, IMapEditFlags, IScaleFlags, IFloorShuffleF
 		public bool? ClampEnemyHpScaling { get; set; } = false;
 		public double EnemyHPScaleFactor { get; set; } = 1;
 		public double BossHPScaleFactor { get; set; } = 1;
+		public bool EnablePoolParty { get; set; } = false;
+		public bool EnableRandomPromotions { get; set; } = false;

```

Comment:
part of your other PR included by accident?

Reply:
noticed that this assumes your other PR is already included, thats fine, this should all resolve once that one is merged in

So you know for the future, you can target your other branch with this PR and when that one is merged in, this one will change to be a PR against alpha


alpha <-- first PR <-- second PR

merge PR 1 and it automatically becomes

alpha <-- second PR

this is pretty recent and its really nice for things like this.